src/__tests__/role-helpers.js

```diff
@@ -0,0 +1,120 @@
+import {getRoles, logRoles} from '../role-helpers'
+
+function setup() {
```

Comment:
Maybe we could use this instead:

```javascript
const section = document.createElement('section')
section.innerHTML = `
  the html for everything else...
`
```

The way it is currently is a bit hard to visualize.

Reply:
Will do.  The only sticking point with doing it that way is that I then have to reach into that dom and pull all those elements out to add them to my expected obj.  But I think the readability of the dom itself is a bigger win, so i agree something like your suggestion should be done.